drivers/ipm/ipm_sedi.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020-2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __DRIVERS_IPM_SEDI_H
+#define __DRIVERS_IPM_SEDI_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "sedi_driver_common.h"
+#include "sedi_driver_ipc.h"
+#include "zephyr/sys/atomic.h"
+
+#define IPC_BUSY_BIT            31
+#define IPM_WRITE_IN_PROC_BIT   0
```

Comment:
add one blank line and a comment line.
they're different thing with the line above them.

Reply:
done